coralproject/cay
# put a background color under the buttons

## What does this PR do?

fixes https://github.com/coralproject/ask/issues/90

adds a white underlay to the Form Field buttons
## How do I test this PR?
- Edit a form
- drag the window until the buttons on the right side of the questions overlap the question text
- it should looks something like this:

![screen shot 2016-09-02 at 11 12 19 am](https://cloud.githubusercontent.com/assets/282503/18212073/3b66ed82-70fe-11e6-8373-261ab9348ee4.png)

@coralproject/frontend
